hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/producer/kafka/KafkaBrokerTopicMetadataFetcher.java

```diff
@@ -0,0 +1,63 @@
+package pl.allegro.tech.hermes.frontend.producer.kafka;
+
+import org.apache.kafka.common.PartitionInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.allegro.tech.hermes.frontend.metric.CachedTopic;
+import pl.allegro.tech.hermes.frontend.producer.BrokerTopicMetadataFetcher;
+
+import java.util.List;
+
+public class KafkaBrokerTopicMetadataFetcher implements BrokerTopicMetadataFetcher {
```

Comment:
I am wondering whether it makes sense to separate `KafkaBrokerTopicMetadataFetcher ` from `KafkaTopicMetadataFetcher`:
1. they names are really similar which makes it confusing
2. KafkaTopicMetadataFetcher is only used by KafkaBrokerTopicMetadataFetcher

Reply:
Good point. Merged.